### ISSUE TITLE
Define evolving content standards

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#21101]
## User story

As a VA.gov writer or designer, I need to know the nuances of the VA.gov voice, tone, and content standards.

## Description

Our content standards have and continue to evolve. We will work with accessibility, IA, and design to continue to define the evolving standards.

This work includes:

- More explicitly defining content standards Beth and Danielle have put in place for plain and compassionate language
- More explicitly defining content patterns and terminology for specific benefits, products, and components
- Preparing content for translation 
- WCAG 2.1 content-specific standards

**Comment by DanielleThierryUSDSVA** [comment]
Hi @1Copenut @artsymartha68! Following up from slack thread. Here's the super epic that's capturing our different streams of content standards work. And here's a parallel content community of practice epic that also captures related work: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9325.

As I mentioned, we'll be reaching out as we make our way through each accessibility-related ticket (and there will be more tickets added). But related to our slack conversation, in the meantime, I have some working notes in this *very, very much WIP and not at all inclusive of all our efforts* notes doc about continuing to make sure we're following preferences around person-first vs. identity-first language. I would welcome any and all thoughts about this. Thanks! https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/content-patterns-and-standards/plain-and-compassionate-language.md

**Comment by DanielleThierryUSDSVA** [comment]
Hi @mpeterz @jenniferlee-dsva @johnhashva @mnorthuis @peggygannon @RLHecht.  We're bringing each of these efforts up for discussion w/ various folks (content/IA, design, accessibility) as we go. But wanted to just tag you in this super epic for general awareness as well. Thanks!
